tests/jupyterlab-cpu/test_rstudio.py

```diff
@@ -0,0 +1,33 @@
+import logging
+import subprocess
+import pytest
+
+LOGGER = logging.getLogger(__name__)
+
+@pytest.mark.parametrize("command,expected_keyword,description", [
+    (
+        "rstudio-server version",
+        "2024.04.2+764 (Chocolate Cosmos) for Ubuntu Jammy",
```

Comment:
Extract Variables such as version out of hardcoded value for ease of maintainability